rpm/udisks2.spec

```diff
@@ -200,7 +201,8 @@ fi
 %{_libdir}/girepository-1.0/UDisks-2.0.typelib
 
 %files -n lib%{name}-devel
-%doc COPYING NEWS README.md AUTHORS HACKING
+%doc NEWS README.md AUTHORS HACKING
```

Comment:
COPYING removal here is one half, the remaining half is adding the "%license COPYING" to the base package. JB#55991

Reply:
Still missing.